### PR TITLE
Add support for properties

### PR DESCRIPTION
**Pull request by cgtobi**
Add support to specify properties to be returned by the methods.

### CODE REVIEW

**Comment by OnFreund** [comment]
@cgtobi can we continue work on this so we can remove the protected accesses from HA?

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@ dist/
 *.egg-info/
 
 venv/
+
```
Comment (OnFreund) [inline .gitignore]:
```suggestion
```

**Comment by OnFreund** [comment]
Other than the empty line added to `.gitignore` this looks great.
I believe it doesn't cover all of the protected access from HA, though.

**Comment by cgtobi** [comment]
There are still a few left. Still working on it.

**Comment by cgtobi** [comment]
The only left is `self._kodi._play_item()`. I don't see much benefit of wrapping that into 4 or more methods just to not use a json string as an argument. Thoughts?

Accompanying HA/core PR: https://github.com/home-assistant/core/pull/40788

**Comment by OnFreund** [comment]
I think it's ok to rename it to `kodi.play_item()` but keep the existing wrappers in place for ease of use.

**Review by OnFreund — APPROVED** [review]

**Comment by OnFreund** [comment]
Thanks!

**Comment by cgtobi** [comment]
My pleasure!

**Pull request merged**